UPGRADE-2.0.md

```diff
@@ -13,6 +11,16 @@
 - The `project` option has been removed.
 - The `extra_data` option has been removed in favour of setting additional data
   directly in the context.
+- The `curl_method` option has been removed in favour of leaving to the user the
+  choice of setting an HTTP client supporting syncronous, asyncronous or both
+  transport methods.
+- The `curl_path` option has been removed.
+- The `curl_ipv4` option has been removed.
+- The `curl_ssl_version` option has been removed.
+- The `verify_ssl` option has been removed.
```

Comment:
We don't have anymore any option related to the HTTP client because each adapter/client has its own set of options, so there is not a replacement for `verify_ssl`. From now on you will have to specify each option supported by the concrete client manually inside the `http_client_options` option.